dags/common/task_group_with_timeout/task_group_with_timeout.py

```
@@ -69,41 +76,74 @@ def __init__(
       child.set_upstream(self._root_node)
-    return super().__exit__(*args)
+
+  def initialize_status_aggregator(self):
+    """Initializes the leaf aggregator task and wires it downstream of
+    in-group leaf children."""
+
+    @task(
+        task_id=self.LEAF_TASK_ID,
+        trigger_rule=TriggerRule.ALL_DONE,
+    )
+    def aggregate_status():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `aggregate_status`

Add return type annotation: `None`

(ANN202)